### PR TITLE
[KO] Apply 11/12 updates to `Tournaments/Official support`

### PR DESCRIPTION
**Pull request by Chronicler01**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Albionthegreat** [comment]
Missing changes from #12417 & #12419

File: wiki/Tournaments/Official_support/ko.md [hunk 1]
```diff
@@ -29,19 +27,35 @@ tags:
 커뮤니티 운영 토너먼트가 공식적인 지원을 받기 위한 조건은 다음과 같습니다:
 
 - 토너먼트 시리즈는 1년에 2번을 넘겨서 열리지 말아야 합니다.
-- 토너먼트는 최소한 16강 더블 엘리미네이션으로 진행되어야 합니다
+- 토너먼트는 최소한 16강 더블 엘리미네이션 또는 32강 싱글 엘리미네이션으로 진행되어야 합니다.
+- LAN 토너먼트는 최소 8강 더블 엘리미네이션 또는 16강 싱글 엘리미네이션으로 진행되어야 합니다.
```
Comment (Albionthegreat) [inline wiki/Tournaments/Official_support/ko.md:31]:
```suggestion
  - LAN 토너먼트는 최소 8강 더블 엘리미네이션 또는 16강 싱글 엘리미네이션으로 진행되어야 합니다.
```
Should be indented.

File: wiki/Tournaments/Official_support/ko.md [hunk 2]
```diff
@@ -119,15 +154,15 @@ tags:
 
 지원 팀의 재량에 따라, 토너먼트에서 공식 지원을 잃게 되는 행위를 한 사람은 최소 4개월 동안 배지가 있는 다른 토너먼트에서 동일한 역할을 하거나 호스트 또는 관리자가 될 수 없습니다. 이 제한이 유저에게 적용될 경우, [토너먼트 팀](mailto:tournaments@ppy.sh)을 통해 유저에게 직접 통보됩니다.
 
-모든 토너먼트 스태프의 명단은 쉽게 접근할 수 있는 위치에 공개되어야 합니다. 이 명단은 토너먼트 포럼 게시물, 웹사이트 혹은 스프레드시트에 포함해야 할 수도 있습니다.
+모든 토너먼트 스태프의 명단은 쉽게 접근할 수 있는 위치에 공개되어야 합니다. 이 명단은 토너먼트 포럼 게시물, 위키 페이지, 웹사이트 혹은 스프레드시트에 포함해야 할 수도 있습니다.
```
Comment (Albionthegreat) [inline wiki/Tournaments/Official_support/ko.md:157]: Missing the "If there are multiple lists of staff, they must all be up to date and in sync with one another." change here.

**Review by Albionthegreat — CHANGES_REQUESTED** [review]
Since I'm not a native speaker, I'm unable to review the language used. Will leave that to sylphi or others.

**Review by TicClick — APPROVED** [review]

**Pull request merged**